### ISSUE TITLE
Unable to run FeGenie.py

### ISSUE THREAD

**Issue by clarelou0128** [Arkadiy-Garber/FeGenie#24]
Hello,

I just downloaded your script (nonconda way) and hoped to use it to search for Fe associated genes in my metagenome. However, I ran into an error when I typed `./FeGenie.py -h`. This same error also occurred when I tried to run the program following your instruction `./FeGenie.py -hmm_lib hmms/iron -bin_dir /directory/of/bins/ -bin_ext fasta -t 16 -out output_fegenie`:

> Traceback (most recent call last):
>   File "./FeGenie.py", line 506, in main
>     test = open(bits)
> FileNotFoundError: [Errno 2] No such file or directory: '/HMM-bitcutoffs.txt'
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "./FeGenie.py", line 3006, in <module>
>     main()
>   File "./FeGenie.py", line 514, in main
>     location = allButTheLast(location, "/")
> UnboundLocalError: local variable 'location' referenced before assignment

Thanks!
Clare

**Comment by Arkadiy-Garber** [comment]
Hi Clare,

Thanks for your interest in using FeGenie.  Have you had a chance to add FeGenie.py into your $PATH? You can do this by editing one of the following files that should be present in your home directory: .bash_profile, .profile, or .bashrc. The periods before the file names are important and makes these files hidden from view unless you specify them directly with a command like 'nano' or 'less'.

If you want to learn how to add FeGenie to your $PATH, you can read more about it here: https://opensource.com/article/17/6/set-path-linux#:~:text=For%20Bash%2C%20you%20simply%20need,bash_profile%2C%20~%2F.

Basically, the FeGenie script 'FeGenie.py' needs to be added to this $PATH in order for the program to run properly.

Thanks, and let me know if you continue to have issues.
Arkadiy

**Comment by clarelou0128** [comment]
This helps, thanks!

Clare
